vm/src/stdlib/os.rs

```diff
@@ -8,6 +8,7 @@ use std::time::{Duration, SystemTime};
 use std::{env, fs};
 
 use crate::crt_fd::Fd;
+use crate::utils::ToCString;
```

Comment:
This unused warning is not resolved

Reply:
Thanks! I did not check clippy in windows. I resolved it 😊